### PR TITLE
[File Blob Export] Returning absolute path of exported file

### PR DESCRIPTION
**Pull request by trantienduchn**
From the conversation here: https://github.com/linagora/james-project/pull/2273#discussion_r269945833

### CODE REVIEW

**Comment by trantienduchn** [comment]
rebased

File: server/blob/blob-export-file/src/test/java/org/apache/james/blob/export/file/LocalFileBlobExportMechanismTest.java [hunk 1]
```diff
@@ -112,9 +112,9 @@ void exportingBlobShouldCreateAFileWithTheCorrespondingContent(FileSystem fileSy
             .element(0)
             .satisfies(sentMail -> {
                 try {
-                    String fileUrl = sentMail.getMsg().getHeader(LocalFileBlobExportMechanism.CORRESPONDING_FILE_HEADER)[0];
+                    String absoluteUrl = sentMail.getMsg().getHeader(LocalFileBlobExportMechanism.CORRESPONDING_FILE_HEADER)[0];
 
-                    assertThat(fileSystem.getResource(fileUrl)).hasSameContentAs(new ByteArrayInputStream(BLOB_CONTENT));
+                    assertThat(fileSystem.getResource("file:///" + absoluteUrl)).hasSameContentAs(new ByteArrayInputStream(BLOB_CONTENT));
```
Comment (rouazana) [inline server/blob/blob-export-file/src/test/java/org/apache/james/blob/export/file/LocalFileBlobExportMechanismTest.java]: does absoluteUrl starts with a "/"? Then you have 4 slashes in the URL?

**Comment by trantienduchn** [comment]
```
[26c5b128ef202957cc20e7aac978bda0317a42a5] [ERROR] Errors: 
[26c5b128ef202957cc20e7aac978bda0317a42a5] [ERROR]   DeletedMessagesVaultTest.vaultExportShouldExportZipContainsVaultMessagesToShareeWhenImapDeleteMessage:491 » FileNotFound
[26c5b128ef202957cc20e7aac978bda0317a42a5] [ERROR]   DeletedMessagesVaultTest.vaultExportShouldExportZipContainsVaultMessagesToShareeWhenImapDeletedMailbox:515 » FileNotFound
[26c5b128ef202957cc20e7aac978bda0317a42a5] [ERROR]   DeletedMessagesVaultTest.vaultExportShouldExportZipContainsVaultMessagesToShareeWhenJmapDeleteMessage:469 » FileNotFound
```
stupid

**Comment by trantienduchn** [comment]
```
[7aa675950da54320330d8e789b245013309c596c] [91mE: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u11_amd64.deb  Could not resolve 'security-cdn.debian.org'
...
```
test this please

**Comment by trantienduchn** [comment]
> IMO we should have two headers:
> 
> * One with the file URL
> * One with the absolute path
> 
> That way, the testing logic is not changed and the admin still have some easy to understand information

It leads to have a guarantee that two headers are identical, then leads to more tests. I propose to keep only the absolute header, and update the tests to make them don't rely on James FileSystem.

**Comment by trantienduchn** [comment]
test this please

**Comment by chibenwa** [comment]
```
[1e5f725f0b4f2985b70d657151168f99e823136d] [ERROR]   ReindexingWithEventDeadLettersTest.indexationShouldBeFailingWhenElasticSearchContainerIsPaused:128 » JsonPath
[1e5f725f0b4f2985b70d657151168f99e823136d] [ERROR]   ReindexingWithEventDeadLettersTest.lambda$redeliverShouldCleanEventDeadLetter$6:149 » JsonPath
```

test this please

**Comment by trantienduchn** [comment]
```
[1e5f725f0b4f2985b70d657151168f99e823136d] [ERROR] Errors: 
[1e5f725f0b4f2985b70d657151168f99e823136d] [ERROR]   SpamAssassinContract.imapMovesToSpamMailboxShouldBeConsideredAsSpam:242 » ConditionTimeout
[1e5f725f0b4f2985b70d657151168f99e823136d] [INFO] 
[1e5f725f0b4f2985b70d657151168f99e823136d] [ERROR] Tests run: 962, Failures: 0, Errors: 1, Skipped: 4
```
test this please

**Comment by chibenwa** [comment]
```
[1e5f725f0b4f2985b70d657151168f99e823136d] [ERROR]   SpamAssassinContract.expungingSpamMessageShouldNotImpactSpamAssassinState:493 » ConditionTimeout
[1e5f725f0b4f2985b70d657151168f99e823136d] [ERROR]   SpamAssassinContract.movingAMailToTrashShouldNotImpactSpamassassinLearning:370 » ConditionTimeout
```

test this please

**Comment by trantienduchn** [comment]
just rebased

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**